app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -441,7 +441,7 @@ def member_subcollections
           results = collection_member_service.available_member_subcollections
           @subcollection_solr_response = results
           @subcollection_docs = results.documents
-          @subcollection_count = results.total
+          @subcollection_count = @presenter.nil? ? 0 : @subcollection_count = @presenter.subcollection_count = results.total
```

Comment:
This would be a really good place for a Nil Presenter. I think it might be a pattern worth considering for general use in cases like this, where a presenter won't be present in update forms. 

https://robots.thoughtbot.com/if-you-gaze-into-nil-nil-gazes-also-into-you